src/java/muuntaja/jackson/DateSerializer.java

```diff
@@ -11,12 +11,20 @@
 public class DateSerializer extends StdSerializer<Date> {
     private final SimpleDateFormat formatter;
 
+    public DateSerializer(String dateFormat) {
+        super(DateSerializer.class, true);
+        formatter = new SimpleDateFormat(dateFormat);
+        formatter.setTimeZone(new SimpleTimeZone(0, "UTC"));
+    }
+
     public DateSerializer() {
         super(DateSerializer.class, true);
```

Comment:
This constructor could now just call the other constructor. `this("yyyy-MM-dd'T'HH:mm:ss'Z'");`